docs/topics/cache.txt

```diff
@@ -381,6 +381,10 @@ behavior. These arguments are provided as additional keys in the
   You can set ``TIMEOUT`` to ``None`` so that, by default, cache keys never
   expire. A value of ``0`` causes keys to immediately expire (effectively
   "don't cache").
+  
+  Not all backends support automatic culling of expired cache entries. When this
+  is the case, the item's timeout value will be triggered when the item is read
+  from the backend.
```

Comment:
The `database` backend removes expired entries when a value is **set** (or `touch()`) in the cache.

I would also move this note that is "database backend"-specific to the [Database caching section](https://docs.djangoproject.com/en/3.2/topics/cache/#database-caching) in docs, e.g.
```diff
diff --git a/docs/topics/cache.txt b/docs/topics/cache.txt
index 068638b382..84be113660 100644
--- a/docs/topics/cache.txt
+++ b/docs/topics/cache.txt
@@ -195,6 +195,9 @@ In this example, the cache table's name is ``my_cache_table``::
         }
     }
 
+The cache doesn't support automatic culling. The expired cache entries are
+culled each time when ``set()``, ``add()``, or ``touch()`` is called.
+
 Creating the cache table
 ~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Thanks. Tweaked and pushed accordingly.